libr/debug/p/debug_gdb.c

```diff
@@ -726,7 +725,6 @@ static const char *r_debug_gdb_reg_profile(RDebug *dbg) {
 #endif
 			);
 		}
-		break;
```

Comment:
Wtf

Reply:
it doesnt hurts because the statements are pretty long, and refactoring or commenting code for testing can become in unexpected results just because the programmer will assume there must be a break on each of these cases. im fine with cases with just return,but if that is a matter of 1-2 lines.